Automattic/jetpack
# E2E tests: Bumped Playwright to 1.10

#### Changes proposed in this Pull Request:

Bumped Playwright to version 1.10.0 and updated the configuration to include `channel` option. 
Also bumped other outdated dependencies just to be up to date.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* CI happy
* To test Playwright channel:
  - In CI, nothing should change, tests run in default Chromium
  - Locally, with Chrome/Edge installed, use `channel: 'chrome'` to run tests in Chrome, and `channel: 'msedge'` to run tests with Edge.
